Loop/Managers/DeviceDataManager.swift

```diff
@@ -151,27 +151,7 @@ class DeviceDataManager: CarbStoreDelegate, TransmitterDelegate, ReceiverDelegat
 
     // MARK: Pump data
 
-    private var latestPumpStatusDate: NSDate?
```

Comment:
This was exactly the same as (and in-sync with) the existing `latestReservoirValue.startDate`, so I just reverted the uses of this one.